### DISCUSSION TITLE
360° projection

### DISCUSSION THREAD

**Discussion by ErickLuis00** [marcofugaro/three-projected-material#38] [Q&A] [answered]
Hi, do you have any idea how I can project an equirectangular image onto the surrounding mesh?

Take a look at this example: https://der-mische.de/panorama/herne-wanne-tunnel/

It's just a 360° panorama that is projected onto the mesh. You can walk around and it looks very realistic.
I'm learning shaders just to do this, but it's hard to understand with so many transformation matrices hahaha

**Comment by marcofugaro** [comment, marked as answer]
Hello! To project onto a sphere you don't need this library, you can just do it in three.js.


Here is the example:  https://threejs.org/examples/?q=env#webgl_panorama_equirectangular

and here is the relative code:

https://github.com/mrdoob/three.js/blob/f0e2b3453f1412b53389beb04add414e3a80023c/examples/webgl_panorama_equirectangular.html#L52-L57
